specificatie/ouder.yaml

```diff
@@ -0,0 +1,61 @@
+openapi: 3.0.3
```

Comment:
Ik neenm aand at deze nog niet is uitgewerkt met een GbaOuder etc....

Reply:
is een GbaOuder nodig? hoe is die anders dan een "gewone" ouder?